src/main/logic/Logic.cpp

```diff
@@ -345,6 +357,8 @@ void Logic::startNewGame(){
 	eventManager->fireEvent(gameStateChangedEvent);
 	//Now we need to initalise everything
 	generateLevel();
+	//Setup the players
+	numLives = 1;
```

Comment:
shouldn't be a problem unless we ever start a game without going through the startNewGame function and this is only read during a running game.